test/library/packages/Python/correctness/paths/checkPaths.chpl

```diff
@@ -0,0 +1,138 @@
+use List, Set, Map;
+use OS.POSIX, Path, FileSystem;
+use Sort;
+use IO;
+use CTypes;
+
+config const verbosePrint = false;
+config const pythonToUse = "python3";
+config const testName: string = "default";
+
+extern proc setenv(name: c_ptrConst(c_char),
+                   envval: c_ptrConst(c_char), overwrite: c_int): c_int;
+
+proc cleanPath(in p) {
+  p = p.strip();
+  if p == "" then return "";
+  if !exists(p) then return "";
+  return realPath(p);
+}
+
+proc getSysPathSpawn() {
+  use Subprocess;
+  var p = spawn(
+    [pythonToUse, "-c", "import sys; print(*sys.path, sep=',')"],
+    stdout=pipeStyle.pipe);
+
+  var output = p.stdout.readAll(string).strip();
+  var paths = new list(string);
+  for p in output.split(",") {
+    var pp = cleanPath(p);
+    if pp != "" && !paths.contains(pp) {
+      paths.pushBack(pp);
+    }
+  }
+  // add '.' to paths
+  var dot = cleanPath(".");
+  if dot != "" && !paths.contains(dot) {
+    paths.pushBack(dot);
+  }
+  return paths;
+}
+proc getSysPathInterpreter() {
+  use Python;
+  var paths = new list(string);
+  {
+    var interp = new Interpreter();
+    var sys = interp.importModule("sys");
+    for p in sys.get(list(string), "path") {
+      var pp = cleanPath(p);
+      if pp != "" && !paths.contains(pp) {
+        paths.pushBack(pp);
+      }
+    }
```

Comment:
Why doesn't this one try to add `.` to the paths?  Is it just always included?

Reply:
When running `python3 -c "import sys; print(*sys.path, sep=',')"`, the current working directory is included in the path as an empty string. This is different than the behavior of running this same code from a file.

But the Chapel module always includes the current directory. So for this test to work, the current directory always needs to be included in the `python -c ....` output

Reply:
Makes sense, thanks!